examples/ibis/models/ibis_full_model_sql.py

```diff
@@ -0,0 +1,26 @@
+import ibis  # type: ignore
+from ibis.expr.operations import Namespace, UnboundTable  # type: ignore
+
+from sqlmesh.core.macros import MacroEvaluator
+from sqlmesh.core.model import model
+
+
+@model("ibis.ibis_full_model_sql", is_sql=True, kind="FULL", audits=["assert_positive_order_ids"])
+
+# This model uses ibis to generate a SQL string.
```

Comment:
are these passed through automatically, or do you have to do description= in the @model tag?

Reply:
oh I didn't even realize you could include a description in the tag, will change